dissect/target/helpers/hashutil.py

```diff
@@ -15,13 +15,9 @@
 
 BUFFER_SIZE = 32768
 
-HashRecord = RecordDescriptor(
-    "filesystem/file/digest",
-    [
-        ("path[]", "paths"),
-        ("digest[]", "digests"),
-    ],
-)
+RECORD_NAME = "filesystem/file/digest"
+NAME_SUFFIXES = ["_resolved_path", "_digest"]
```

Comment:
```suggestion
NAME_SUFFIXES = ["_resolved", "_digest"]
```

I'd still rename this, as a lot of filed names are or end with `"path"`, you'd get `path_resolved_path`, which doesn't look nice.